src/main/io/displayport_max7456.c

```diff
@@ -125,6 +125,19 @@ static uint32_t txBytesFree(const displayPort_t *displayPort)
     return UINT32_MAX;
 }
 
+static textAttributes_t supportedTextAttributes(const displayPort_t *displayPort)
+{
+    UNUSED(displayPort);
+
+    textAttributes_t attr = TEXT_ATTRIBUTES_NONE;
+    TEXT_ATTRIBUTES_ADD_INVERTED(attr);
+    TEXT_ATTRIBUTES_ADD_SOLID_BG(attr);
+    if (!displayConfig()->force_sw_blink) {
```

Comment:
In general drivers shouldn't use PGs. We need to pass the required configuration to `max7456Init` via `vcdProfile_t`.